src/experimental/define-config.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { defineConfig } from './define-config'
+
+describe('defineConfig', () => {
+  it('should return the config as-is when no future option is provided', () => {
+    const config = {
+      preset: 'ts-jest',
+      testEnvironment: 'node',
+    }
+
+    const result = defineConfig(config)
+
+    expect(result).toBe(config)
```

Comment:
```suggestion
    expect(defineConfig(config)).toEqual({
        testEnvironment: 'node'
    })
```

Please use this way of writing tests for the rest of this file. The assertion statement should always show the details of the expected value, not via a constant